src/trpc/router/userCreationRequestRouter.ts

```diff
@@ -47,6 +48,41 @@ export const userCreationRequestRouter = router({
         return creationRequest;
     }),
 
+    edit: protectedProcedure.input(editUserCreationRequestSchema).mutation(async ({ input, ctx }) => {
+        accessCheck(checkRoleForAccess(ctx.session.user.role, 'editUserCreationRequests'));
```

Comment:
Should we check if request is denied here?

Reply:
good idea, thanks

Reply:
added check of request state below